src/test_driver/openiotsdk/integration-tests/all-clusters-app/test_app.py

```diff
@@ -2,14 +2,14 @@
 #    Copyright (c) 2023 Project CHIP Authors
 #    All rights reserved.
 #
-#    Licensed under the Apache License, Version 2.0 (the "License");
+#    Licensed under the Apache License, Version 2.0 (the 'License');
```

Comment:
I'd rather not change the header format. It's in accordance with the Matter pattern.